rpc/src/cluster_tpu_info.rs

```diff
@@ -36,6 +37,7 @@ impl TpuInfo for ClusterTpuInfo {
             .cluster_info
             .tpu_peers()
             .into_iter()
+            .chain(once(self.cluster_info.my_contact_info()))
```

Comment:
seems `ClusterInfo::tpu_peers()` is only used by `ClusterTpuInfo`. so, moving this logic into the called method is a possibility. However, various `ClusterInfo::*_peers()` methods are consistent with excluding its own `ContactInfo`. So, I guess placing this logic here isn't terribly wrong. ;)